supertokens_python/recipe/session/recipe_implementation.py

```diff
@@ -39,20 +40,33 @@
     AccessTokenObj,
     RecipeInterface,
     RegenerateAccessTokenOkResult,
+    SessionClaim,
+    SessionClaimValidator,
     SessionInformationResult,
     SessionObj,
+    ValidateClaimsOkResult,
+    SessionDoesntExistError,
+    JSONObject,
+    GetClaimValueOkResult,
 )
 from .session_class import Session
+from ...types import MaybeAwaitable
+from .utils import (
+    SessionConfig,
+    update_claims_in_payload_if_needed,
+    validate_claims_in_payload,
+)
 
 if TYPE_CHECKING:
     from typing import List, Union
 
     from supertokens_python.querier import Querier
 
-    from .utils import SessionConfig
 
 from .interfaces import SessionContainer
 
+_T = TypeVar("_T")
```

Comment:
doesn't seem to be used anywhere

Reply:
Nice catch.. How do you manage to find used vars so easily? Some tool? 

Removed.